Test/src/main/CalculatorDateAndTime.java

```diff
@@ -0,0 +1,18 @@
+package main;
+
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+import java.util.TimeZone;
+
+public class CalculatorDateAndTime {
+
+    public static String toString(ZonedDateTime date) { return "";}
+
+    public static ZonedDateTime calculate(ZonedDateTime date, String operation) {return ZonedDateTime.of(2018,
+            6, 26, 0, 0, 0, 0,
+            ZoneId.of(TimeZone.getDefault().getID()));}
```

Comment:
if statement do not fit in one line then dont start it right after bracket, do it as usual
method {
   long_statement;
}